### DISCUSSION TITLE
Gokapi LXC

### DISCUSSION THREAD

**Discussion by bub4** [tteck/Proxmox#2417] [Ideas]
Hi @tteck 
first of all, please let me express how thankful I am for your great work! Really appreciate your efforts.

Gokapi looks like a great application:
https://github.com/Forceu/Gokapi

What do you think?

cheers

**Comment by bub4** [comment]
Hi @tteck 
updating this lxc does return: ✗ There is currently no update path available.
I am on v 1.8.0
https://github.com/Forceu/Gokapi/releases

Thank you so much

**Comment by DimonXe** [comment]
Same problem
